ledger/controllers/wallet.js

```diff
@@ -290,7 +293,7 @@ const write = function (runtime, apiVersion) {
         return reply(boom.resourceGone('cannot perform a contribution using a legacy surveyor'))
       } else {
         // new contribution surveyor not yet populated with voting surveyors
-        const errMsg = 'surveyor ' + surveyor.surveyorId + ' has 0 surveyors, but needed ' + votes
```

Comment:
`votes` here will always be undefined.  do we want to calculate votes earlier then we do currently?

Reply:
I defined `totalVotes` earlier than this block so it should always be defined now https://github.com/brave-intl/bat-ledger/pull/516/commits/6d4a5bdb0f29d9c372298aa65eb50be7cba2aaa9